src/BlazorStrap/Components/Model/BSModal.razor.cs

```diff
@@ -33,13 +43,26 @@ protected override async Task OnAfterRenderAsync()
                 await new BlazorStrapInterop(JSRuntime).FocusElement(Me);
                 JustOpened = false;
             }
+            for (int i = 0; i < EventQue.Count; i++)
+            {
+                await EventQue[i].InvokeAsync(BSModalEvent);
+                EventQue.RemoveAt(i);
+            }
         }
 
         protected string styles
         {
             get
             {
-                var display = _isOpen ? "display: block; padding-right: 17px;" : null;
+                var display = "";
```

Comment:
We should change this to use CssBuilder - although simple, it will match the rest

Reply:
We need to look into Modals soon. We have some formatting issues. That should be easy to solve.